src/schema/artwork/meta.js

```diff
@@ -15,8 +15,9 @@ const forSaleIndication = artwork =>
 
 const dimensions = artwork => artwork.dimensions[artwork.metric]
 
-const partnerDescription = ({ partner: { name }, forsale }) => {
-  if (!name) return undefined
+const partnerDescription = ({ partner, forsale }) => {
+  const name = partner && partner.name
+  if (isEmpty(name)) return undefined
```

Comment:
Also! Any reason to return `undefined` here instead of `null`?

Reply:
No real reason! Either should work, so I just left it as-is, it's just a helper method used by the resolver.